config/chime_science_run_gpu.yaml

```diff
@@ -499,6 +499,25 @@ baseband:
     kotekan_stage: basebandReadout
     in_buf: network_buffer_3
 
+# Bad inputs buffer
+bad_input_buffer:
+  num_frames: buffer_depth
+  frame_size: num_elements
+  metadata_pool: main_pool
+  bad_inputs_buffer:
+    kotekan_buffer: standard
+
+# Buffer bad input updates
+# 1) Reorder list
+# 2) Zero bad inputs mask
+# 3) Add current bad input mask
+buffer_bad_input_update:
+  updatable_config:
+    bad_inputs: /updatable_config/bad_inputs
+  buffer_bad_inputs: 
+    kotekan_stage: bufferBadInputs
+    out_buf: bad_inputs_buffer
```

Comment:
This could be restructured like this:
```
  buffer_bad_inputs: 
    kotekan_stage: bufferBadInputs
    updatable_config:
      bad_inputs: /updatable_config/bad_inputs
    out_buf: bad_inputs_buffer
```